src/components/Table/Table.scss

```diff
@@ -5,12 +5,13 @@
 // --------------------------------------------------
 // Data table styles
 
-
 .ms-Table {
   display: table;
   width: 100%;
+  border-collapse: collapse;
 }
 
+.ms-Table tr,
 .ms-Table-row {
   display: table-row;
   height: 30px;
```

Comment:
Looks like line-height being set to 30px is making the computed row height 31px. I'm not sure if we need the line-height in there but if not, we should remove it.